indexer/packages/postgres/src/types/db-model-types.ts

```diff
@@ -264,6 +264,11 @@ export interface SubaccountUsernamesFromDatabase {
   subaccountId: string,
 }
 
+export interface AddressUsernameFromDatabase {
```

Comment:
We only use `FromDatabase` to denote that it's the exact object from postgres, lets just store this as `AddressUsername`

Reply:
Sounds good.